scripts/trace-next-server.js

```diff
@@ -52,6 +48,16 @@ async function main() {
     },
   })
 
+  // remove temporary package packs
+  pkgPaths.forEach((packagePath) => {
+    fs.unlinkSync(packagePath)
+  })
+  // remove changes to package.json files from packing
+  await execa('git', ['checkout', '.'], {
```

Comment:
Should it be `package.json` instead of `.`?

Reply:
Would that remove changes to all `package.json` files in the tree or just the top-level one? This currently modifies all `packages/*/package.json` files to ensure we link to the packed versions correctly and then we want to undo those changes here after we're done installing

Reply:
I see. then it could be `git checkout packages/*/package.json` but I suppose theres no harm in checking out everything.

Also if there are additional files created you could do `git clean -fd`